src/sentry/feedback/blueprints/api.md

```diff
@@ -0,0 +1,303 @@
+| release           | string           |                                                     |
+| replay_id         | optional[string] | The id of a running replay.                         |
+| sdk               | object           | SDK information.                                    |
+| sdk.name          | string           | -                                                   |
+| sdk.version       | string           | -                                                   |
+| status            | string           | One of: resolved, unresolved.                       |
+| tags              | object           | Mapping of key, value pairs.                        |
+| timestamp         | string           | ISO-8061 formatted UTC datetime.                    |
+| url               | string           | The page the user-feedback was triggered on.        |
+| user              | object           | The authorized user's information.                  |
+| user.display_name | optional[string] | -                                                   |
+| user.email        | optional[string] | -                                                   |
+| user.id           | optional[string] | -                                                   |
+| user.ip           | optional[string] | Same search field as Events                         |
+| user.username     | optional[string] | -                                                   |
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "browser": {
+          "name": "Chome",
+          "version": "103.0.38"
+        },
+        "contact_email": "colton.allen@sentry.io",
+        "device": {
+          "brand": "Apple",
+          "family": "iPhone",
+          "model": "11",
+          "name": "iPhone 11"
+        },
+        "dist": "abc123",
+        "environment": "production",
+        "id": "1ffe0775ac0f4417aed9de36d9f6f8dc",
+        "locale": {
+          "lang": "en",
+          "timezone": "UTC+1"
+        },
+        "message": "I really like this user-feedback feature!",
+        "os": {
+          "name": "iOS",
+          "version": "16.2"
+        },
+        "platform": "javascript",
+        "release": "version@1.3",
+        "replay_id": "ec3b4dc8b79f417596f7a1aa4fcca5d2",
+        "sdk": {
+          "name": "sentry.javascript.react",
+          "version": "6.18.1"
+        },
+        "status": "unresolved",
+        "tags": {
+          "hello": "is",
+          "it": ["me", "you're", "looking", "for"]
+        },
+        "timestamp": "2023-08-31T14:10:34.954048",
+        "url": "https://docs.sentry.io/platforms/javascript/",
+        "user": {
+          "display_name": "John Doe",
+          "email": "john.doe@example.com",
+          "id": "30246326",
+          "ip": "213.164.1.114",
+          "username": "John Doe"
+        }
+      }
+    ]
+  }
+  ```
+
+### Create User Feedback [POST]
```

Comment:
could we move this POST under its own section? since the API URL is now just `/feedback/` (no org slug)